zerver/lib/timestamp.py

```diff
@@ -44,4 +43,4 @@ def datetime_to_timestamp(dt: datetime.datetime) -> int:
 
 def datetime_to_precise_timestamp(dt: datetime.datetime) -> float:
     verify_UTC(dt)
-    return time.mktime(dt.timetuple()) + dt.microsecond/1000000
+    return dt.timestamp()
```

Comment:
The `verify_UTC` is unnecessary; `.timestamp()` works fine with timezones. And then this is not worth having as a helper function—the caller should just use `.timestamp()` directly.

Reply:
I'd also wonder whether we actually want microsecond precision here when we don't use it elsewhere.